bootstrap.py

```diff
@@ -1383,4 +1410,15 @@ def exec_service(service, command, failonerror=True):
             remove_obsolete_packages()
 
         if options.remote_exec:
-            install_foreman_ssh_key()
+            if options.remote_exec_proxies:
+                listproxies = options.remote_exec_proxies.split(",")
+                for proxy_fqdn in listproxies:
+                    remote_exec_url = "https://" + str(proxy_fqdn) + ":9090/ssh/pubkey"
```

Comment:
Not great if we ever move around the smart proxy port or someone configures it differently. Fine to keep, worth noting.